docs/neopixel.rst

```diff
@@ -0,0 +1,69 @@
+Neopixel
+****
+
+.. py:module:: neopixel
+
+The ``neopixel`` module lets you use Neopixel (WS2812) individually addressable RGB LED strips with the Microbit.    
+Note to use the ``neopixel`` module, you need to import it separately with::
+
+    import neopixel
+
+.. warning::
+    
+    Do not use the 3v connector on the Microbit to power any more than 8 Neopixels at a time.
```

Comment:
I've successfully used the 3v connector on the micro:bit to drive 30 pixels. Out of interest, what's the worst that could happen..?